lib/cmds/ontologies_cmds/import.js

```diff
@@ -27,35 +29,37 @@ exports.handler = async argv => {
   const project = argv.project;
   const path = `/v1/terminology/projects/${project}/relationships/`;
 
-  let values = [];
+  let buffer = [];
   const pipeline = chain([
     stdin(),
     withParser(),
     async (data) => {
       const value = data.value;
 
       if (Array.isArray(value)) {
-        values = values.concat(value);
+        buffer = buffer.concat(value);
       } else {
-        values.push(value);
+        buffer.push(value);
       }
 
-      if (values.length >= batch) {
-        const payload = values.slice();
-        values = [];
-        const response = await patch(argv, path, payload);
-        print(response.data, argv);
+      if (buffer.length >= batch) {
+        const chunks = chunk(buffer, batch); // split buffer into chunks of size batch
```

Comment:
maybe consider renaming to `_chunk` to not confuse with the `chunk` in the loop below.